src/AzureDevOpsPolicyConfigurator/Interfaces/IConnectionProvider.cs

```diff
@@ -0,0 +1,18 @@
+﻿using AzureDevOpsPolicyConfigurator.Arguments;
```

Comment:
I would suggest to have interfaces in the same namespace as the implementation to avoid unecessary usings